en-US/macros.lang

```diff
@@ -0,0 +1,146 @@
+## NOTE: https://www-demo5.allizom.org/
```

Comment:
Drop this file, and add to main.lang strings that are not available from the [new piece of code](https://github.com/jpetto/bedrock/blob/4a4da9d750695cc0c592690773d9874aa43438c5/bedrock/base/templates/macros.html#L410-L441).

Reply:
I will add the strings to main.lang. I did see strings available in many files, and was thinking about using your script to leverage them. Ok. I will move new strings there.